content/rs/installing-upgrading/uninstalling.md

```diff
@@ -6,27 +6,12 @@ alwaysopen: false
 categories: ["RS"]
 aliases: /rs/administering/installing-upgrading/uninstalling/
```

Comment:
We need to update two more pages: [Upgrading](https://docs.redislabs.com/latest/rs/installing-upgrading/upgrading/), and [cluster-recovery ](https://docs.redislabs.com/latest/rs/administering/troubleshooting/cluster-recovery/).

Both must use same the directories paths and user/group.

Reply:
Acc to Gidon upgrade doesn't use the paths and user. I updated cluster recovery.